### ISSUE TITLE
PWA Spanish updates to footer links and menu labels, and privacy page language.

### ISSUE THREAD

**Issue by depietrodeanna** [episphere/connect#1105]
The NIH website team received a comment from a confused visitor about the Certificate of Confidentiality link on the Privacy webpage on MyConnect: https://myconnect.cancer.gov/#privacy. 

They said that perhaps because we use the word, “our,” the visitor mistakenly thought the link would lead them to Connect’s particular Certificate.

Instead, let's use this suggestion:
"Maintain a Certificate of Confidentiality (https://grants.nih.gov/policy/humansubjects/coc.htm) from the National Institutes of Health. Learn more about Certificates of Confidentiality, including how the Certificate will help protect against any legal requests (such as a court order) to give out information that could identify you."

The edits are small and I think we should make them. We have other MyConnect updates that are minor but should be addressed for Spanish consistency. Adding those for next-in whenever the Dev team has bandwidth to address. These edits should go on the Privacy page on MyConnect and I will update the public website page in conjunction. 

Here are the two discrepancies that we’re seeing between MyConnect and Spanish Public site:

1) Inconsistent Main Menu labels – the Spanish labels for the four linked pages are different on MyConnect vs the public site.

Public site: 
<img width="593" alt="Screenshot 2024-08-29 at 7 57 41 AM" src="https://github.com/user-attachments/assets/6270d178-c955-4a45-ab74-8ad81dc5fc4d">

MyConnect:  
<img width="650" alt="Screenshot 2024-08-29 at 7 57 05 AM" src="https://github.com/user-attachments/assets/3f45ec61-93e4-4232-9c0b-be562e2b44c3">

We should make make the MyConnect labels match the public site labels.

2) MyConnect Footer links have different listed text than their counterparts on the public site, they also point to English URLs vs Spanish URLs. 

Here are the five Spanish URLs they should point to:

Accesibilidad: https://www.cancer.gov/espanol/politicas/accesibilidad
Descargo de: https://www.cancer.gov/espanol/politicas/exoneracion
FOIA: https://www.cancer.gov/espanol/politicas/foia
Privacidad: https://www.cancer.gov/espanol/politicas/confidencialidad-seguridad
Divulgacion de: [https://www.hhs.gov/es/vulnerability-disclosure-policy/index.html](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.hhs.gov%2Fes%2Fvulnerability-disclosure-policy%2Findex.html&data=05%7C02%7Cdeanna.depietro%40nih.gov%7Cdc1d5ec8fd69427347fb08dcc3875386%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C638600233198983192%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=RVn1JH%2FTz%2FTsBoKLhU7HmPLZSNi256xw98xRFpNn7kI%3D&reserved=0)

**Comment by robertsamm** [comment]
@brotzmanmj @Davinkjohnson same here. Do we have an est. date for when we want to do any 'Spanish clean up' items that we could include this item for?

**Comment by bransteitterbr** [comment]
@depietrodeanna  and @robertsamm This is in dev and ready for testing

**Comment by depietrodeanna** [comment]
Tested in Dev and looks good, ready for Stage @bransteitterbr

**Comment by sonyekere** [comment]
Hi @depietrodeanna is this ready for prod?

**Comment by depietrodeanna** [comment]
yes, sorry for not indicating. this has been tested and is good in stage. @sonyekere

**Comment by jwiernusz** [comment]
As built
